src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newObject = {};
+
+  for (const [key, value] of Object.entries(robot)) {
```

Comment:
Using 'Object.entries()' in a for loop could cause the function to slow down, especially if the input object is large. Consider using 'Object.keys()' or 'Object.values()' to optimize performance.

Reply:
Or you can use loop `for in`